src/main/java/bio/terra/service/filedata/flight/ingest/ValidateIngestFileAzureDirectoryStep.java

```diff
@@ -66,9 +73,4 @@ public StepResult doStep(FlightContext context) throws InterruptedException {
 
     return StepResult.getStepResultSuccess();
   }
-
```

Comment:
Given the NPE on line 65 (```} else if (!existingEntry.getLoadTag().equals(loadModel.getLoadTag())) {```) we should be defensive

Reply:
This was caused by a bug where we weren't writing a loadTag out to Azure file Storage Table entries. This is now fixed, so I don't think there's a need to be all that defensive here. Maybe we should add a test that tries to ingest the same file twice and check for the correct failure.